opba-api-security/src/main/java/de/adorsys/opba/api/security/domain/OperationType.java

```diff
@@ -0,0 +1,7 @@
+package de.adorsys.opba.api.security.domain;
+
+public enum OperationType {
+    AIS,
+    BANK_SEARCH,
+    TOKEN
```

Comment:
This is not TOKEN but rather CONFIRM_CONSENT

Reply:
Renamed